AdyenDelegatedAuthentication/AdyenDelegatedAuthentication.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mohamed Eldoheiri on 14/11/2023.
+//
```

Comment:
Header :)

Reply:
there're quite a bit of those in the project actually, maybe it should be a separate PR to fix those :-)